sycl/test-e2e/Graph/Explicit/while_recording.cpp

```diff
@@ -31,8 +28,19 @@ int main() {
       Success = true;
     }
   }
+  assert(Success);
 
-  Graph.end_recording();
+  Success = false;
+  try {
+    Graph.add({});
+  } catch (sycl::exception &E) {
+    auto StdErrc = E.code().value();
+    if (StdErrc == static_cast<int>(errc::invalid)) {
+      Success = true;
+    }
```

Comment:
```suggestion
    Success = E.code() == static_cast<int>(errc::invalid)
```
A similar change was made in the E2E test upstream PR, so might as well sort this one here as well.